wherehows-web/app/components/more-info.ts

```diff
@@ -0,0 +1,41 @@
+import Component from '@ember/component';
+
+export default class MoreInfo extends Component {
+  tagName = 'span';
+
+  classNames = ['more-info'];
+
+  /**
+   * Proxies to anchor element target attribute
+   * @type {string}
+   * @memberOf MoreInfo
+   */
+  target: string;
+
+  /**
+   * Proxies to anchor element href attribute
+   * @type {string}
+   * @memberOf MoreInfo
+   */
+  link: string;
+
+  /**
+   * Renders the tool tip component, if present
+   * @type {string}
+   * @memberOf MoreInfo
+   */
+  tooltip: string;
+
+  constructor() {
+    super(...arguments);
+
+    this.target || (this.target = '_blank');
```

Comment:
It saves having to write a constructor function if you just do 
`link: string = this.link || '#'` in the class definition instead

Reply:
I've done this before, but I think this might be a TS bug that works, because this also doesn't seem to be documented. I'll confirm